tree/ntuple/v7/test/ntuple_print.cxx

```diff
@@ -60,7 +64,9 @@ TEST(RNtuplePrint, FullString)
        + "* N-Tuple : Staff                                                              *\n"
        + "* Entries : 1                                                                  *\n"
        + "********************************************************************************\n"
-       + "* Field 1   : pt (float)                                                       *\n"
+       + "* Field 1   : px (float)                                                       *\n"
```

Comment:
Are field accessible by number?

Reply:
The fields are ordered but there is currently no API to access them by (this) number.  There is a field ID, generally different from the number printed in `PrintInfo()`, which is unique per ntuple. The field ID is accessible through the field descriptor.

Reply:
Fair enough.  So it does not matter whethere this print routine start at Field 0 or 1 :)